theia/api/models/imagery_request.py

```diff
@@ -34,6 +34,10 @@ class ImageryRequest(models.Model):
     project = models.ForeignKey(Project, related_name='imagery_requests', on_delete=models.CASCADE)
     pipeline = models.ForeignKey(Pipeline, related_name='imagery_requests', on_delete=models.CASCADE)
 
+    bearer_token = models.CharField(max_length=2048, default="tist")
```

Comment:
It's kind of hard to tell--does this actually store the tokens in the db as fields on an imagery_request table? Or are these just built in memory, per-request?